coding/python_testing.rst

```diff
@@ -29,7 +28,7 @@ A simple :mod:`unittest` example is shown below:
 
 The important things to note in this example are:
 
-* Test file names must begin with ``test_`` for maximum compatibility with `pytest`_.
+* Test file names must begin with ``test_`` to allow `pytest`_ to automatically detect them without requiring an explicit list of tests be supplied.
```

Comment:
"without requiring an explicit test list, which can be hard to maintain and can lead to missed tests." might provide better rationale.